docs-src/references/dynamic-configuration.md

```diff
@@ -123,18 +124,20 @@ The Rate Per Second (RPS) dynamic configuration keys set the rate at which reque
+| `internal-frontend.globalNamespaceRPS` | Int  | Namespace rate limit per second across all internal-frontends.                                                                                                                                                                                                                                                                             | 0             |
+| History                                |      |                                                                                                                                                                                                                                                                                                                                            |               |
+| `history.rps`                          | Int  | Request rate per second for each History host.                                                                                                                                                                                                                                                                                             | 3000          |
+| Matching                               |      |                                                                                                                                                                                                                                                                                                                                            |               |
+| `matching.rps`                         | Int  | Request rate per second for each Matching host                                                                                                                                                                                                                                                                                             | 1200          |
+| `matching.numTaskqueueWritePartitions` | Int  | Number of read partitions for a Task Queue. Note that to increase the number of partitions (on a running cluster), first increase `numTaskqueueReadPartitions` and then `numTaskqueueWritePartitions`. To decrease the number of partitions, first decrease `numTaskqueueWritePartitions` and then decrease `numTaskqueueReadPartitions`.  | 4             |
+| `matching.numTaskqueueWritePartitions` | Int  | Number of write partitions for a Task Queue. Note that to increase the number of partitions (on a running cluster), first increase `numTaskqueueReadPartitions` and then `numTaskqueueWritePartitions`. To decrease the number of partitions, first decrease `numTaskqueueWritePartitions` and then decrease `numTaskqueueReadPartitions`. | 4             |
```

Comment:
don't duplicate the instructions twice? also, I'm not sure if we should include those instructions here at all? they're subtly wrong: you have to decrease write partitions _and then_ make sure all spooled tasks are processed

Reply:
yeh; I think these should be better explained and this is not the place for it.